tools/distpartitioning/parmetis_wrapper.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import logging
+import os
+import platform
+import sys
+from pathlib import Path
+
+import constants
+from utils import read_json
+
+
+def run_parmetis_wrapper(params):
+    """Function to execute all the steps needed to run ParMETIS
+
+    Parameters:
+    -----------
+    params : argparser object
+        an instance of argparser class to capture command-line arguments
+    """
+    assert os.path.isfile(params.schema_file)
+    assert os.path.isfile(params.hostfile)
+
+    schema = read_json(params.schema_file)
+    graph_name = schema[constants.STR_GRAPH_NAME]
+    num_partitions = len(schema[constants.STR_NUM_NODES_PER_CHUNK][0])
+
+    # Trigger pre-processing step to generate input files for ParMETIS.
+    preproc_cmd = (
+        f"mpirun -np {num_partitions} -hostfile {params.hostfile} "
+        f"python3 tools/distpartitioning/parmetis_preprocess.py "
```

Comment:
how about looking for xxx_process.py via `os.path.dirname(os.path.abspath(__file__))`?